frontend/src/layout/navigation/TopNavigation.tsx

```diff
@@ -96,6 +99,30 @@ export function TopNavigation(): JSX.Element {
                 </div>
             </div>
             <div className="text-center mt" style={{ paddingRight: 16, paddingLeft: 16 }}>
+                {preflight?.cloud && billing?.should_display_current_bill && (
+                    <Link to="/organization/billing" data-attr="top-menu-billing-usage">
+                        <Card
+                            bodyStyle={{ padding: 4, fontWeight: 'bold' }}
+                            style={{ marginBottom: 16, cursor: 'pointer' }}
+                        >
+                            <span className="text-small text-muted">
+                                <b>Current usage</b>
+                            </span>
+                            <div style={{ fontSize: '1.05rem' }}>
+                                {billing?.current_bill_amount !== undefined && billing?.current_bill_amount !== null ? (
+                                    `$${billing?.current_bill_amount?.toLocaleString()}`
+                                ) : (
+                                    <>
+                                        Unavailable{' '}
+                                        <Tooltip title="We can't show your current bill amount right now. If you keep seeing this message, contact us.">
```

Comment:
Can we be a little more descriptive? I think the only scenario in which we expect this is when Stripe hasn't calculated the billing yet, is that correct? So maybe we can say something closer to "Your invoice has not been calculated yet. Invoice totals are computed daily. If this message has not updated within 24 hours, please contact us."

Reply:
Actually no, if the bill hasn't been calculated this will show $0. This just happens when the Stripe API is unavailable or there is some connection/request error.